pkg/asset/manifests/template.go

```diff
@@ -33,6 +33,7 @@ type OpenStackCredsSecretData struct {
 // VSphereCredsSecretData holds encoded credentials and is used to generated cloud-creds secret
 type VSphereCredsSecretData struct {
 	VCenter              string
+	Port 				 string
```

Comment:
AFAIK we do not need to add port to the credentials. We need to add port to the `global` and `workspace` sections of the cloud provider config: https://github.com/openshift/installer/blob/master/pkg/asset/manifests/vsphere/cloudproviderconfig.go